index.js

```diff
@@ -1,8 +1,22 @@
 const config = {
+    env: {
+        es6: true,
+        jest: true,
+        browser: true,
+        node: true
+    },
     plugins: ['jsx-a11y', 'package-json', 'react', 'react-hooks'],
-    extends: ['plugin:jsx-a11y/recommended', 'plugin:package-json/recommended'],
+    extends: [
+        'eslint:recommended',
+        'prettier',
+        'plugin:jsx-a11y/recommended',
+        'plugin:package-json/recommended'
+    ],
     rules: {
+        'getter-return': 'off',
```

Comment:
I don't think we should turn anything `off`. Would rather we use `warnings` or leave them as default.

Reply:
Lint warnings are an antipattern IMO. It's either a problem or it's not. change_my_mind.jpeg

Reply:
+1 on avoiding using warnings.

re: `no-empty` and `getter-return`, those both seem like rules that would be likely to catch real bugs.

I think the situations you'd want to avoid those rules (empty catch block, side-effect-y getter) you'd want a code comment to explain anyway, so maybe we let those cases be `eslint-ignore`'d inline?

Reply:
If we are against warning then leave them on :D